src/main/java/StringManipulation.java

```diff
@@ -1,32 +1,128 @@
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+/**
+ * The implementation class StringManipulation for the StringManipulationInterface.
+ *
+ * @author DeWayne Dantzler dewayne.dantler@bellevuecollege.edu
+ */
 public class StringManipulation implements StringManipulationInterface {
-
+	private String _MyString;
```

Comment:
Depending on the coding convention, some say variables should start with lowercase letter, so having an underscore as the 1st char allows the 2nd one to be uppercase. Also, the underscore can be a visual queue that this is an instance/field variable rather than local variable.

Reply:
I see, thanks for clarifying.